c2cgeoportal/views/layers.py

```diff
@@ -362,7 +376,16 @@ def metadata(self):
         if not layer.public and self.request.user is None:
             raise HTTPForbidden()
 
-        return self._metadata(str(layer.geo_table), layer.exclude_properties)
+        # exclude the columns used to record the last features update
+        exclude = layer.exclude_properties if layer.exclude_properties else ""
+        last_update_date = self.settings.get("last_update_date_column", False)
+        if last_update_date:
+            exclude += "," + last_update_date
```

Comment:
`exclude` can be ',date'?
the start ',' is wrong...
I think that `exclude` should be a list and at the end we should do `",".join(exclude)` ...